### ISSUE TITLE
Images are missing alt text

### ISSUE THREAD

**Issue by Shadlock0133** [zedeus/nitter#559]
Alt text is send by the twitter api, but isn't preserved by nitter. It's just a matter of propagating it in the code. I've implemented bare-bones version of it here: https://github.com/Shadlock0133/nitter/commit/df0551ef9ba14f444eac9c0b748bad750e1739b6

**Comment by chocmake** [comment]
Would be quite useful. Have been noticing quite a number of users take advantage of the feature for storing the text of neural network prompts (eg: [these](https://twitter.com/proximasan/status/1552969931641294848) still life images of asparagus).
